routes/users.js

```diff
@@ -0,0 +1,9 @@
+var express = require('express');
+var router = express.Router();
+
+/* GET users listing. */
+router.get('/', function(req, res, next) {
+  res.send('respond with a resource');
+});
+
+module.exports = router;
```

Comment:
This file is unnecessary. I would challenge you to make an express app next week or in the future without express-gen so you can get a better understanding of how all the inner parts of express connect together. :)